### DISCUSSION TITLE
Thoughts about Google's new Manifest V3 announcements

### DISCUSSION THREAD

**Discussion by achtchaern** [Eloston/ungoogled-chromium#2626] [General]
Since Google has made new [announcements about the upcoming Manifest V3 enforcement](https://developer.chrome.com/blog/resuming-the-transition-to-mv3?hl=en), I've wondered if and how the UG authors / community are going to react to this change.. Sure, Google has been repeatedly softening up the actual policies from MV3, but I think it's still clear that the Adblockers are not going to be of the same quality once MV2 gets phased out.

If I had to guess, I'd say that there are not alot of UG users who don't use an Adblocker. If UG just continue to comply with the announced upstream change, the UG user base is probably going to shrink substantially..

What's your opinion about this? Are you going to comply with the MV3 enforcement? Or do you have no choice at all?

**Comment by PF4Public** [comment]
Please [use search first](https://github.com/ungoogled-software/ungoogled-chromium/issues?q=is%3Aissue+Manifest+v2).

**Reply by achtchaern** [reply]
There isn't any recent statement to be found the thread you linked. But I see, it doesn't make sense to create a new discussion. Sorry
